src/redux/Books/books.js

```diff
@@ -0,0 +1,17 @@
+const ADD = 'bookstore/books/ADD';
+const REMOVE = 'bookstore/books/REMOVE';
+
+const initialState = [];
+
```

Comment:
- [ ] Great job so far. Please ensure that you Create and Export Action Creators for book like you did for categories (ADD and REMOVE functions)